pom.xml

```diff
@@ -1188,6 +1188,8 @@
 
     <module>arquillian</module>
 
+    <module>fractions/health-api</module>
```

Comment:
yeah, it should be moved. and I will also restore the previous package location so it remains compatible. 

I am with a top level `apis` module, but would suggest to move the `jaxrs-client-api` separately (i.e. create a new Jira for it.)